test/webui/components/PackageSidebar/infos.spec.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Infos component
+ */
+
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import Infos from '../../../../src/webui/src/components/PackageSidebar/modules/Infos';
+import { packageMeta } from '../store/packageMeta';
+ 
+console.error = jest.fn();
+
+describe('<PackageSidebar /> : <Infos />', () => {
+  it('should load the component and check getter: info with package data', () => {
+    const wrapper = mount(<Infos packageMeta={packageMeta} />);
+    const instance = wrapper.instance();
+    const result = {
```

Comment:
Just being a bit picky here. 
https://codecov.io/gh/verdaccio/verdaccio/pull/757/diff
Perhaps we should add different scenarios here as missing homepage, repo or license. But in a new PR if you wish.

Reply:
That's probably a good idea. How should I go about serving different version of meta data? Deep copy `packageMeta` and modify it would be one way I guess. If that's ok, is there a copy function in the test framework available or should I utilize lodash?